faunadb-scala/src/load/scala/faunadb/FaunaClientLoadSpec.scala

```diff
@@ -187,6 +187,25 @@ class FaunaClientLoadSpec extends FixtureAsyncWordSpec with Matchers with ScalaF
         succeed
       }
     }
+
+    "fail to handle more than 100 concurrent streams on the same client" in { client =>
+      val maxConcurrentStreamCount = 100
+      // create collection
+      val collectionName = RandomGenerator.aRandomString
+      val setup = for {
+        _ <- client.query(CreateCollection(Obj("name" -> collectionName)))
+        createdDoc <- client.query(Create(Collection(collectionName), Obj("credentials" -> Obj("password" -> "abcdefg"))))
+        docRef = createdDoc("ref")
+        // create a first publisher to setup the connection that will be reused by all the other publishers (makes 101 streams)
+        _ <- client.stream(docRef)
+        publisherValues <- Future.traverse(List.fill(maxConcurrentStreamCount)(docRef))(ref => client.stream(ref))
```

Comment:
is `100` the limit? or `> 100` the limit? because the `List.fill(..)` will generate only 100 streams

Reply:
`100` is the max valid value indeed.

However I am opening an additional stream the line above to unsure that the connection already exists, thus making the total 101.

I'll try to rework the test to make it clearer :+1: